Update xcm integrations template issue (#542)
Manta-Network/Manta: .github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md

        5. `DepositAsset: { assets: Wild { Wild: All }, maxAssets: 1, beneficiary: { parents: 0, interior: X1 { X1: AccountId32 { network: Any, id: SovereignAccountonRelay } } } }`
    
    **Note:** The values used above are for reference to be used in this testing environment, do not use these values in production!

> 	**Note:** to watch out for having enough funds for the `WithdrawAsset` instruction and to reserve in the encoded HRMP calls.

Reviewer: ```suggestion
	**Note:** Verify you have enough funds to execute the `WithdrawAsset` instruction **and** to reserve the required amount encoded in the HRMP calls.
```
not sure why github is rendering this more indented than the L116 note, it looks a bit weird in the review md

Author: Thanks.
Because it's tabs vs spaces. I think spaces are better, at least the last formatting problem was because of tabs again so I stopped using them here.

> 	**Note:** to ensure the `Transact`'s `requireWeightAtMost` is appropriate for the encoded call, especially if you are batching multiple HRMP extrinsics for example.

Reviewer: ```suggestion
	**Note:** Ensure the `Transact`'s `requireWeightAtMost` is appropriate for the encoded call, especially if you are batching multiple HRMP extrinsics.
```

Author: thx

Name: Dolphin
Symbol: DOL
Decimals: 18
Min Balance: 1
Self sufficient: true
```

> - Note: Calamari MultiLocation ad metadata are different!

Reviewer: ```suggestion
- Note: Calamari MultiLocation and metadata are different!
```

Author: thx
